sw/device/silicon_creator/lib/base/mock_abs_mmio.h

```diff
@@ -0,0 +1,112 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BASE_MOCK_ABS_MMIO_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BASE_MOCK_ABS_MMIO_H_
+
+#include "sw/device/lib/base/testing/mock_mmio_test_utils.h"
+#include "sw/device/lib/testing/mask_rom_test.h"
+#include "sw/device/silicon_creator/lib/base/abs_mmio.h"
+
+namespace mask_rom_test {
+namespace internal {
+/**
+ * Mock class for abs_mmio.c.
+ */
+class MockAbsMmio {
+ public:
+  MOCK_METHOD(uint32_t, Read8, (uint32_t addr));
+  MOCK_METHOD(void, Write8, (uint32_t addr, uint32_t value));
+  MOCK_METHOD(uint32_t, Read32, (uint32_t addr));
+  MOCK_METHOD(void, Write32, (uint32_t addr, uint32_t value));
+
+  virtual ~MockAbsMmio() {}
+
+ protected:
+  ::testing::InSequence seq_;
```

Comment:
I don't think this belongs in the mock. `MaskRomTest` already has [this](https://github.com/lowRISC/opentitan/blob/165ddec05a4d2795fee832aac580ff281308d1d7/sw/device/lib/testing/mask_rom_test.h#L79).

Reply:
Removed `InSequence` from `MockAbsMmio`.